packages/sdk/src/messages/NomadMessage.ts

```diff
@@ -447,7 +448,12 @@ export class NomadMessage<T extends NomadContext> {
    * status of the message.
    */
   async replicaStatus(): Promise<ReplicaMessageStatus> {
-    return this.replica.messages(this.leaf);
+    const root = await this.replica.messages(this.leaf);
```

Comment:
will this still work before we perform the upgrade?

do we need
```
    if (root === ethers.constants.HashZero || root === 0) return ReplicaMessageStatus.None;
    if (root === `0x${'00'.repeat(31)}02` || root == 2)
      return ReplicaMessageStatus.Processed;
```

Reply:
fixed